core/shared/src/main/scala/scalaz/zio/Promise.scala

```diff
@@ -20,40 +20,37 @@ import java.util.concurrent.atomic.AtomicReference
 import scalaz.zio.internal.Executor
 import Promise.internal._
 
-/**
- * A promise represents an asynchronous variable that can be set exactly once,
- * with the ability for an arbitrary number of fibers to suspend (by calling
- * `get`) and automatically resume when the variable is set.
+/** A promise represents an asynchronous variable that can be set exactly once,
```

Comment:
I'd add a newline right after `/**`, because well-formatted Scaladoc will start on the next line, after the initial `/**`.